src/utils/auth/preferencesMapper.ts

```diff
@@ -40,6 +40,7 @@ const getPreferenceGroupValue = (
     return {
       reciter: DEFAULT_XSTATE_INITIAL_STATE.reciterId,
       playbackRate: DEFAULT_XSTATE_INITIAL_STATE.playbackRate,
+      volume: DEFAULT_XSTATE_INITIAL_STATE.volume,
```

Comment:
I don't think we should store `volume` remotely since volume preferences can change per user per device. So we should remove this line as it's only used for remotely stored settings.